### DISCUSSION TITLE
React 18.2

### DISCUSSION THREAD

**Discussion by vadimshvetsov** [TanStack/query#4096] [Ideas] [answered]
What do you think about using `^18` in react and react-dom peerDeps instead of `^18.0.0` in react-query package?

Right now when trying to install it with npm without `--force` or `--legacy-peer-deps` with 18.2 React it refutes to install but technically works with this version of React.

**Comment by TkDodo** [comment, marked as answer]
Those should be equivalent? `^18.0.0` means "compatible with 18.0.0", and `18.2` is compatible with that. You can check that by going to https://semver.npmjs.com/, picking `react` and then enter the range `^18.0.0`.

It's a bit obfuscated with all the `experimental` releases, but you can see `18.0.0`, `18.1.0` and `18.2.0` being marked as green, which means they match this dependency.
